Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -268,6 +273,11 @@ local questsToBlacklistBySoDPhase = {
         [90135] = true, -- Hiding Rogue Quick Draw Teldrassil for now as there are too many icons
         [90136] = true, -- Hiding Rogue Quick Draw Durotar for now as there are too many icons
         [90137] = true, -- Hiding Rogue Quick Draw Tirisfal Glades for now as there are too many icons
+        [90139] = true, -- Hiding Warrior Quick Strike Loch Modan for now as there are too many icons
+        [90140] = true, -- Hiding Warrior Quick Strike Elwynn Forest for now as there are too many icons
+        [90141] = true, -- Hiding Warrior Quick Strike Darkshore for now as there are too many icons
+        [90142] = true, -- Hiding Warrior Quick Strike The Barrens for now as there are too many icons
```

Comment:
Same as with the Darkshore quest. Reduce the spawns of https://www.wowhead.com/classic/npc=209797/bruuz to a single one and we don't need to blacklist this quest.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5435/commits/1beb0ba3bc28881ab492b5c55447be712cec2a44

Reply:
```suggestion
```